### ISSUE TITLE
Advice on malware domains

### ISSUE THREAD

**Issue by JeremyWW** [uBlock-LLC/uBlock#700]
Hope it's OK to ask in this way...

On the basis that too many filter lists in use is not a good thing, i.e. potential browsing slow downs, I'm seeking opinion on the best of the malware domain lists to use. No point in having huge overlaps.

Thanks.

**Comment by gorhill** [comment]
There is no overlap, duplicates are discarded at parse time.

**Comment by JeremyWW** [comment]
OK, any performance hit though for using too many lists in uBlock?

**Comment by gorhill** [comment]
I really never tested with _all_ the lists, but so far no, I do not have worries. My only warning would be to just use common sense. Somebody reportedly [fed uBlock with nearly 950K](https://github.com/gorhill/uBlock/issues/27#issuecomment-67652255) entries and apparently it worked all fine, but then uBlock is optimized to deal well with plain hostname filters.

**Comment by JeremyWW** [comment]
I agree about the common sense approach which is why I'd rather have the minimum number of lists in use. Problem is I can't decide on the malware lists! I can see how the overlaps are discarded now. The numbers against each list tell you that.

**Comment by gorhill** [comment]
Really I don't have any better insights as to which one is better.

**Comment by JeremyWW** [comment]
OK, thanks. I did have one problem with the Dan Pollock filter yesterday and persisting today - any attempt to update the filter results in a warning from Kaspersky AV and the update is blocked. See attached. I've told Dan - he tells me this happens occasionally but isn't sure what he can do about it so I've flagged it with Kaspersky themselves and here.

![2015-02-06_110501](https://cloud.githubusercontent.com/assets/544340/6078452/9ce1b042-adf0-11e4-8f63-e3a23d0a1972.png)
